### ISSUE TITLE
Cannot change database when there are jails present

### ISSUE THREAD

**Issue by szepeviktor** [fail2ban/fail2ban#1072]
Could you explain this message?
```
# modified courier-smtp.conf to courier-smtp.local
$ fail2ban-client reload courier-smtp
ERROR  NOK: ('Cannot change database when there are jails present',)
```
v0.9.1-1

**Comment by leeclemens** [comment]
Had you changed the value of `dbfile` since the last restart/reload?

**Comment by szepeviktor** [comment]
Ofcourse not.

**Comment by sebres** [comment]
It's a bug, will fix it...

**Comment by sebres** [comment]
closed with #1077

**Comment by szepeviktor** [comment]
Thank you.
